client/hwlib/src/collectors/hardware_info.rs

```diff
@@ -265,7 +264,10 @@ mod tests {
             get_test_filepath("cpuinfo_max_freq").as_path(),
         ))
         .unwrap();
-        assert_eq!(processor.codename, "Tiger Lake");
+        assert_eq!(
+            processor.identifier,
+            [0xc1, 0x6, 0x8, 0x0, 0xff, 0xfb, 0xeb, 0xbf]
```

Comment:
Just to avoid magic constants, I think it could be nice to add a comment here noting that this is the Tiger Lake ID. (I assume this is the Tiger Lake ID)

Reply:
Added in bb02da2